modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -92,15 +89,19 @@ describe('browser.tinymce.core.content.EditorContentTest', () => {
   // TODO: TINY-8367 The table plugin code has been moved to core so there is now always an extra DIV in the dom for the table resize bars
   // Safari differs in behaviour compared to the other browsers when getting text content for inline mode.
   // When the resize bar div is included in the DOM, editor.getBody().innerText includes two extra \n at the end
-  it('TINY-6281: getContent text', () => testGetTextContent('<p>Text to be retrieved</p>', 'Text to be retrieved' + (isSafari() ? '\n\n' : '')));
+  it('TINY-6281: getContent text', () => testGetTextContent('<p>Text to be retrieved</p>', 'Text to be retrieved'));
 
   it('TINY-8578: getContent text, empty line in div', () => testGetTextContent('<div><p></p></div>', ''));
 
   it('TINY-8578: getContent text, empty line', () => testGetTextContent('<p></p>', ''));
 
-  it('TINY-8578: getContent text, two empty lines in div', () => testGetTextContent('<div><p></p><p></p></div>', isSafari() ? '\n\n' : '\n\n\n\n'));
+  it('TINY-8578: getContent text, two empty lines in div', () => testGetTextContent('<div><p></p><p></p></div>', '\n\n'));
+
+  it('TINY-8578: getContent text, repeating two empty lines in divs', () => testGetTextContent('<div><p></p><p></p></div><div><p></p><p></p></div>', '\n\n\n\n\n\n'));
+
+  it('TINY-8578: getContent text, two empty lines', () => testGetTextContent('<p></p><p></p>', '\n\n'));
 
-  it('TINY-8578: getContent text, two empty lines', () => testGetTextContent('<p></p><p></p>', isSafari() ? '\n\n' : '\n\n\n\n'));
+  it('TINY-8578: getContent text with bogus element', () => testGetTextContent(`<p data-mce-bogus="all">content</p><p></p>`, '', 'raw'));
```

Comment:
I wonder if these tests should be in the `EditorGetContentTextFormatTest` test file instead?

Also do we have tests that check the iframe output as well as inline? It looks like this only tests inline.

- [ ] Ensure tests exist for both iframe and inline modes